public/blog/chinese-web-release.md

```diff
@@ -0,0 +1,12 @@
+The Chinese version of Crosswalk website has been released! The structure of Chinese website is similar with the English version, mainly consisting of four parts: [文档](https://crosswalk-project.org/documentation/getting_started_zh.html), [博客](https://crosswalk-project.org/blog/), [贡献](https://crosswalk-project.org/contribute/index_zh.html) and [维基](https://github.com/crosswalk-project/crosswalk-website/wiki). To accomplish the swift of language version, you are
+required to click the language button at the top right of the page and then select the language you wish.
```

Comment:
Chinese characters in the English version is hard to understand for us unenlightened readers.